src/components/Inputs/input.module.scss

```diff
@@ -1,7 +1,8 @@
 .input-wrapper {
+    --input-border-color: var(--grey-color-60);
```

Comment:
Now that the default is `--grey-color-60`, you'll need to create a second local variable, perhaps name it `--input-border-color-active` it should be `--grey-color-80`. Update all the active and hover border states to use this so we don't lose those states. These should be around the following lines:

339,
347,
1042,
1173,
1188

Reply:
@dkilgore-eightfold fixed